docker/Dockerfile

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu:16.04
+MAINTAINER prozlach@mesosphere.io
```

Comment:
https://docs.python.org/3.6/faq/library.html#what-kinds-of-global-value-mutation-are-thread-safe